package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@copy/v86",
```

Comment:
I'd suggest calling it v86 and syncing with the repository later.

cc @giulioz, any thoughts? Is this something you can rebase your repo onto?